itou/siaes/management/commands/move_siae_data.py

```diff
@@ -230,18 +250,26 @@ def handle(self, from_id, to_id, wet_run=False, only_job_applications=False, **o
                 )
 
         self.stdout.write(
-            "MOVE {} OF siae.id={} FINISHED\n".format("DATA" if move_all_data else "JOB APPLICATIONS", from_siae.pk)
+            "MOVE {} OF siae.id={} FINISHED\n".format(
+                "DATA" if move_all_data else "JOB APPLICATIONS AND EMPLOYEE RECORDS", from_siae.pk
```

Comment:
Le bout de code `"DATA" if move_all_data else "JOB APPLICATIONS AND EMPLOYEE RECORDS"` est présent à l'identique au moins deux fois dans ce script, c'est peut-être l'occasion de le factoriser ?

Reply:
Faudrait :grin:, mais je ne voulais pas tirer ce bout de ficelle aujourd'hui car ça commence par ça puis faut changer le type de formatage des `str`, puis on vois qu'on peux factoriser d'autre trucs, mais que les tests ne couvrent pas tout, donc on en écrit, et on est 2 jours plus tard :older_adult:.

Reply:
Elle est collector ta réponse ! Je la réutiliserais dans mes PR. 🤣